docs/it/token_endpoint.rst

```diff
@@ -224,6 +224,11 @@ ID Token
 ++++++++
 
 L'ID Token è un JSON Web Token (JWT) che contiene informazioni sull'utente che ha eseguito l'autenticazione. I RP DEVONO eseguire la validazione dell'ID Token.
+
+.. admonition:: |cieid-icon|
+
+  Il RP PUÒ richiedere che L'ID Token sia cifrato (vedere il parametro *id_token_encrypted_response_alg* nel :ref:`Metadata RP <MetadataRP>` ). In questo caso l'ID Token sarà un JWT **firmato e cifrato** e DEVE contenere il parametro *cty* (Content-Type) nell'intestazione JOSE con il valore *JWT* (vedere :rfc:`7515#section-4`).
```

Comment:
ref

https://datatracker.ietf.org/doc/html/rfc7519#section-5.2

Reply:
fixed in https://github.com/italia/spid-cie-oidc-docs/pull/55/commits/8ec33eca96a7075ce9c222eb16e07b2fc6820176